shell/gpu/gpu_surface_gl.cc

```diff
@@ -87,7 +89,7 @@ static SkColorType FirstSupportedColorType(GrContext* context, GLenum* format) {
     *format = (y);                                 \
     return (x);                                    \
   }
-#if OS_MACOSX && !OS_IOS
+#if (OS_MACOSX && !OS_IOS) || OS_LINUX
```

Comment:
I'm assuming OS_LINUX isn't true for OS_ANDROID?

Reply:
yes - OS_LINUX is only set for host Linux